sources/packages/backend/libs/integrations/src/institution-integration/ier12-integration/utils-service/application-event-code-during-enrolment-and-completed-utils.service.ts

```diff
@@ -92,20 +95,36 @@ export class ApplicationEventCodeDuringEnrolmentAndCompletedUtilsService {
       : ApplicationEventCode.DISS;
   }
 
+  /**
+   * Checks if there is any full-time disbursement feedback errors.
+   * @param disbursementFeedbackErrors disbursement feedback errors.
+   * @returns true, if there is any full-time disbursement feedback
+   * errors.
+   */
+  private hasFullTimeDisbursementFeedbackErrors(
+    disbursementFeedbackErrors?: DisbursementFeedbackErrors[],
+  ): boolean {
+    return disbursementFeedbackErrors?.some((disbursementFeedbackError) =>
```

Comment:
Is the null check needed here?
If it is, is the method going to return only `boolean`?